db/seeds/dev/songs.js

```diff
@@ -0,0 +1,32 @@
+
+exports.seed = function (knex, Promise) {
+  return knex('playlistSongs').del()
+    .then(() => knex('playlists').del())
+    .then(() => knex('songs').del())
+    .then(() => {
+      return Promise.all([
+        knex('songs').insert({ name: 'Bohemian Rhapsody', artist_name: 'Queen', genre: 'Rock', song_rating: 100 }, 'id')
+          .then(song1 => {
+            return knex('songs').insert({ name: 'Another One Bites the Dust', artist_name: 'Queen', genre: 'Rock', song_rating: 85 }, 'id')
+            .then(song2 => {
+              return knex('playlists').insert({playlist_name: 'Workout Songs'}, 'id')
+              .then(playlist1 => {
+                return knex('playlists').insert({playlist_name: 'Wedding Songs'}, 'id')
+                .then(playlist2 => {
+                  return knex('playlistSongs').insert({song_id: song1[0], playlist_id: playlist1[0]})
+                  .then(() => {
+                    return knex('playlistSongs').insert({song_id: song2[0], playlist_id: playlist2[0]})
+                    .then(() => {
+                      return knex('playlistSongs').insert({song_id: song2[0], playlist_id: playlist1[0]})
+                    })
```

Comment:
This seems like it could get really hairy if you had even 20 seeds, just in terms of the indentations, I wonder if you would break up the seeds into groups depending on the table, or if you could just not indent.  Actually, I'm thinking indentation might make this file less readable rather than easier to read if we were to add more seeds.  Just some thoughts if we decide to add more, we can get rid of the indentations.